framework/dask-distributed/Dockerfile

```
@@ -0,0 +1,23 @@
+
+# Install and configure build prerequisites.
+RUN apt-get update && apt-get install --yes git
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /usr/local/bin/uv
+ENV UV_COMPILE_BYTECODE="true"
+ENV UV_LINK_MODE="copy"
+ENV UV_SYSTEM_PYTHON="true"
+
+# Install payload package.
+ADD requirements.txt /
+RUN \
+    --mount=type="cache",id="uv-${TARGETARCH}${TARGETVARIANT}",target="/root/.cache/uv" \
```

Hadolint (2.14.0)

[error] unexpected ""cache"
expecting "bind", "cache", "secret", "ssh", or "tmpfs"

(DL1000)